### PR TITLE
Docs: add a document for network fault simulation

### PR DESCRIPTION
**Pull request by en-jin19**
Signed-off-by: en-jin19 <jinenwei@pingcap.com>

In this PR, I have added a document about simulating process faults

This PR is translated from: https://github.com/chaos-mesh/website-zh/pull/35

### CODE REVIEW

File: docs/simulate-network-chaos-in-physical-nodes.md [hunk 1]
```diff
@@ -1,6 +1,288 @@
 ---
```
Comment (en-jin19) [inline docs/simulate-network-chaos-in-physical-nodes.md:1]: 哈喽王老师，麻烦也请帮忙看下这篇译文，谢谢！ @WangXiangUSTC

File: docs/simulate-network-chaos-in-physical-nodes.md [hunk 2]
```diff
@@ -1,6 +1,288 @@
...
+
+| Configuration item | Abbreviation | Description                                                                                                                 | Value                                                                                                                     |
+|:------------------ |:------------ |:--------------------------------------------------------------------------------------------------------------------------- |:------------------------------------------------------------------------------------------------------------------------- |
+| correlation        | c            | The correlation between the percentage of current corrupt occurrence and the previous occurrence.                           | Int. It is a percentage ranging from 0 to 100 (10 is 10%) ("0" by default ).                                              |
+| device             | d            | Name of the impacted network interface card                                                                                 | String, such as “eth0”. The value is mandatory.                                                                           |
```
Comment (WangXiangUSTC) [inline docs/simulate-network-chaos-in-physical-nodes.md:83]: maybe `required` is more common than `mandatory`

File: docs/simulate-network-chaos-in-physical-nodes.md [hunk 3]
```diff
@@ -1,6 +1,288 @@
...
+
+| Configuration item | Abbreviation | Description                                                                                                                 | Value                                                                                                                     |
+|:------------------ |:------------ |:--------------------------------------------------------------------------------------------------------------------------- |:------------------------------------------------------------------------------------------------------------------------- |
+| correlation        | c            | The correlation between the current latency and the previous one.                                                           | Int. It is a percentage ranging from 0 to 100 (10 is 10%) ("0" by default).                                               |
+| device             | d            | The name of impacted network interface card                                                                                 | String, such as “eth0”. The value must be set.                                                                            |
```
Comment (WangXiangUSTC) [inline docs/simulate-network-chaos-in-physical-nodes.md:127]: the description of `device` is different from the above, it's better to use the same.

**Comment by en-jin19** [comment]
I have revised the translations according to the review, please kindly take a look, thanks! @WangXiangUSTC

File: docs/simulate-network-chaos-in-physical-nodes.md [hunk 4]
```diff
@@ -1,6 +1,288 @@
...
+title: Simulate Network Faults
+sidebar_label: Simulate Network Faults
 ---
 
-## TODO
+This document introduces how to use Chaosd to simulate network faults. The simulations can be completed by modifying network routing and traffic flow control using iptables, ipsets, tc, etc.
+
+## Notes
+
+Make sure the NET_SCH_NETEM module is installed in the Linux kernel. If you are using CentOS, you can install the module through the kernel-modules-extra package. Most other Linux distributions have installed it already by default.
```
Comment (cwen0) [inline docs/simulate-network-chaos-in-physical-nodes.md:10]: `NET_SCH_NETEM`

**Review by WangXiangUSTC — APPROVED** [review]
LGTM

**Pull request merged**